esmvaltool/diag_scripts/validation.py

```diff
@@ -340,14 +340,15 @@ def main(cfg):
     for short_name in grouped_input_data:
         logger.info("Processing variable %s", short_name)
 
-        cmip_era = cfg["cmip_era"]
+        # cmip era does not have to be required here #nocmip
+        # cmip_era = cfg["cmip_era"]
         # get the control, experiment and obs dicts
         ctrl, exper, obs = get_control_exper_obs(short_name, input_data, cfg,
-                                                 cmip_era)
+                                                 cmip_type=cfg["cmip_era"] if
+                                                 "cmip_era" in cfg else None)
```

Comment:
it's safer to set the kwarg outside the function call, also is `cmip_type` a positional argument now or just a kwarg?

Reply:
It is optional and has a default value. Selection per project is not needed in my case, as I apply it solely to EMAC simulations (not cmip era applicable). Maybe you have a better application in mind to decide if this approach is reasonable?

Reply:
no that's fine, I was just concerned about explicitly setting a positional arg when it's not positional - I'll have another look now :+1:

Reply:
yep it is indeed a positional :+1: